setup.py

```diff
@@ -36,6 +40,48 @@ def run(self):
         ]
         return install_data.run(self)
 
+def win_get_llvm_reg():
+    REG_PATH = "SOFTWARE\\LLVM\\LLVM"
+    try:
+      return winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE, REG_PATH, 0, winreg.KEY_READ | winreg.KEY_WOW64_32KEY)
+    except FileNotFoundError:
+      pass
+    return winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE, "SOFTWARE\\LLVM\\LLVM", 0, winreg.KEY_READ) 
```

Comment:
Maybe we can use the variable `REG_PATH` here?

Reply:
Yes, copy/paste is a terrible feature :)